packages/hardhat-kadena/src/plugin.ts

```diff
@@ -129,6 +133,7 @@ extendConfig((config, userConfig) => {
           precompiles: {
             chainwebChainId: CHAIN_ID_ADDRESS,
             spvVerify: VERIFY_ADDRESS,
+            create2Factory: CREATE2_FACTORY_ADDRESS,
```

Comment:
This needs to be documented in packages/hardhat-kadena/README.md in several places.